gobrew.go

```diff
@@ -331,7 +330,8 @@ func (gb *GoBrew) CurrentVersion() string {
 // Uninstall the given version of go
 func (gb *GoBrew) Uninstall(version string) {
 	if version == "" {
-		log.Fatal("[Error] No version provided")
+		utils.Errorln("[Error] No version provided")
+		os.Exit(1)
```

Comment:
We can create new function in utils and use it here.
what do you think?

Reply:
like:
```go
func Fatalln(a ...interface{}) {
	_, _ = ColorError.Println(a...)
        os.Exit(1)
}
```

Reply:
Will change em all then